internal/service/networkcontainer/resource_network_container_test.go

```diff
@@ -57,6 +57,41 @@ func TestAccNetworkContainerRS_basicAWS(t *testing.T) {
 	})
 }
 
+func TestAccNetworkContainerRS_updateCidrWithoutChangingRegions(t *testing.T) {
+	var (
+		randIntUpdated   = acctest.RandIntRange(0, 255)
+		cidrBlockUpdated = fmt.Sprintf("10.8.%d.0/24", randIntUpdated)
+		projectName      = acc.RandomProjectName()
```

Comment:
Tagging @lantoli since recently working on our test infra to see if something is advisable here

Reply:
thanks @BrettFolkins , I'll look into this and will probably create a new PR based on yours so I can do some small changes